jitbuilder/build/files/host/x.mk

```diff
@@ -18,7 +18,4 @@
 # [2] http://openjdk.java.net/legal/assembly-exception.html
 #
 # SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
-###############################################################################
-
-JIT_PRODUCT_BACKEND_SOURCES+=\
-    $(JIT_PRODUCT_DIR)/x/runtime/AsmUtil64.asm
+###############################################################################
```

Comment:
Rather than delete this content completely (leaving no local example of what to do in this file) could you delete the file name but leave `JIT_PRODUCT_BACKEND_SOURCES+=` commented out ?

Reply:
Please change the 2nd copyright date above to 2016, 2017